data_storage/src/elastic_handler.py

```diff
@@ -0,0 +1,18 @@
+from elasticsearch import Elasticsearch
+class ElasticsearchHandler:
+    """Handles interactions with Elasticsearch."""
+    
+    def __init__(self, elasticsearch_url):
+        self.es = Elasticsearch([elasticsearch_url])
+        self.index_name = "kafka_messages"  
+    def index_message(self, message):
+        """Indexes a message into Elasticsearch."""
+        document = {
```

Comment:
Nw do końca jak działa elastic, ale chyba nie powinniśmy indeksować danych związanych z kafka? key, partition, offset

Reply:
+1.  To bedzie zmieniane w ramach kolejnego zadania dot. elastika na nowym branchu